src/Tes.Runner/Docker/DockerExecutor.cs

```diff
@@ -177,13 +179,11 @@ private async Task PullImageWithRetriesAsync(string imageName, string? tag, Auth
         {
             logger.LogInformation(@"Pulling image name: {ImageName} image tag: {ImageTag}", imageName, tag);
 
-            await dockerPullRetryPolicy.ExecuteWithRetryAsync(async () =>
-            {
-                await dockerClient.Images.CreateImageAsync(
+            await dockerPullRetryPolicy.ExecuteWithRetryAsync(
```

Comment:
are we sure this is equivalent to the previous?

Reply:
Other than removing a large compiler-provided state machine, slowing performance for no gain (other than possibly a slightly better exception call stack), this is equivalent to the previous.